tasks/core/apps.yml

```diff
@@ -41,24 +41,22 @@
 - name: Install external apps
   block:
     - name: Find full nextcloud version
-      command: ./occ -V
+      command: ./occ status  --output=json
       args:
         chdir: "{{ nextcloud_installation_dir }}"
       become: true
       become_user: "{{ nextcloud_file_owner }}"
-      register: nextcloud_full_version
+      register: _output
```

Comment:
Let's use the `_result` variable for consistency, I have used that in a couple of places in the role